resources/oe/src/oeCore.i

```diff
@@ -403,6 +432,16 @@ FUNCTION GET_MODE RETURNS CHARACTER ():
     RETURN inputObject:GetJsonObject("params"):GetCharacter("mode").
 END FUNCTION.
 
+FUNCTION IS_RECID_OR_ROWID RETURNS CHARACTER (cTableName AS CHARACTER, cColumnKey AS CHARACTER):
```

Comment:
`IS_RECID_OR_ROWID` suggests that it returns a boolean value, but it returns a string. My suggestion is to change the name to `GET_FORMATTED_COLUMN` or something like that.